### DISCUSSION TITLE
Mounting Solution

### DISCUSSION THREAD

**Discussion by Addonnis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#183] [Ideas]
With the control panel off, I noticed some additional unused mounting holes.

I tried 3D printing a cover so the ESP8266 could sit inside, and affix underneath.

I haven't designed a 3D model before so it could use a lot of improvement. The 'tabs' either side broke off when I tried to screw them down, but I think the idea has potential.

![Example_01](https://user-images.githubusercontent.com/97511696/149613267-66d47450-c6a9-431e-aec3-4b9d06dcca4c.jpg)
![Example_02](https://user-images.githubusercontent.com/97511696/149613265-a9adf341-50ba-499d-9ab9-cd03e38c2649.jpg)

**Comment by chunkysteveo** [comment]
That's a great idea!

**Comment by torei** [comment]
cool, you know about https://www.thingiverse.com/ ?
you may want to upload your thing :]
